source/Background/Controller.js

```diff
@@ -186,6 +208,14 @@ backgroundController.exposeController(
           status: CONNECTION_STATUS.pending,
           icon: icons[0] || null,
           timeout,
+          date,
+          events: [
+            ...response?.[keyring.currentWalletId]?.apps[domainUrl].events,
+            {
+              status: CONNECTION_STATUS.pending,
+              date,
+            }
+          ],
```

Comment:
Do we want to log this in the events?

Reply:
idk, i put it there just to keep consistency, but we dont really use it

Reply:
ill remove it